CODE/VulnScan/v3/_train.py

```
@@ -0,0 +1,391 @@
+                                                  random_state=42)
+
+# Train Model
+try:
+    train_model(model_name=MODEL_NAME,
+                epochs=EPOCHS,
+                batch_size=BATCH_SIZE,
+                learning_rate=LEARN_RATE,
+                save_model_path=SAVE_PATH,
+                use_cuda=CUDA)
+except FileNotFoundError as e:
+    logger.error(f"File Not Found Error in training model: {e}")
```

Ruff (0.8.0)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)